include/dpp/emoji.h

```diff
@@ -71,22 +71,11 @@ class DPP_EXPORT emoji : public managed, public json_interface<emoji> {
 	json to_json_impl(bool with_id = false) const;
 
 public:
-	/**
-	 * @brief Emoji name
-	 */
-	std::string name{};
-	/**
-	 * @brief User id who uploaded the emoji
-	 */
-	snowflake user_id{0};
-	/**
-	 * @brief Flags for the emoji from dpp::emoji_flags
-	 */
-	uint8_t flags{0};
-	/**
-	 * @brief Image data for the emoji if uploading
-	 */
-	std::string image_data{};
+	std::string 			name{};		//!< emoji name
```

Comment:
Why change the documentation formatting here?

Reply:
yeah probably best not to, because we're getting rid of all the `//!<` formatting junk

Reply:
this was already agreed upon in discord that we would ignore this and sort it in a later PR.

Reply:
But why does your PR change the style?
This is *your* PR introducing the style we want to stop using, this sounds like the perfect time to revert this change